assets/styles/informed.scss

```diff
@@ -2095,3 +2095,23 @@ caption {
   background-color: $error-colour;
   border-color: $error-colour;
 }
+
+.contents-links {
+  font-size: 1rem;
+  line-height: 1.75;
+  list-style-type: none;
```

Comment:
would `list-style-type: "—";` be ok? (then you can get rid of the __wrapper)

Reply:
Fair point, I just copied this style from the GOV site but I'll try it out.